tests/vmi_multus_test.go

```diff
@@ -186,11 +191,12 @@ var _ = Describe("Multus", func() {
 				Expect(err).ToNot(HaveOccurred())
 				tests.WaitUntilVMIReady(detachedVMI, tests.LoggedInCirrosExpecter)
 
-				pingVirtualMachine(detachedVMI, "10.1.1.1", "\\$ ")
+				err := testPingFromVMI(detachedVMI, testIP, "\\$ ")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s should be reachable", detachedVMI.Name)
```

Comment:
1. The test `Describe/Context/` should contain the desired behaviour (commonly start/contain the word "should" ). Here an unexpected error occurred, the text should contain the error. For example `VMI %s failed to ping IP %s, detachedVMI.Name, testIP`.
2. Anyway, here you say the vmi should be reachable. Does getting an error from `testPingFromVMI` means the vmi is not reachable? You're pinging the `testIP` not the vmi.